UKHomeOffice/nmsw-ui
# NMSW-1293 Update template page with new content and templates

# Ticket



----

## AC

Update tab templates page to "Voyage report forms"
Provide the templates for download for other FAL forms

----

## To test

- check page matches wireframe attached (note wireframe is missing the need help section)
- check need help section content is same as on prototype
- check new FAL downloads work

----

## Notes
